UI/Navigation/src/Navigation/Presentation/PageNavigation/PageNavigation.xaml

```diff
@@ -0,0 +1,42 @@
+﻿<Page x:Class="Navigation.Presentation.PageNavigation"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:local="using:Navigation.Presentation"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:uen="using:Uno.Extensions.Navigation.UI"
+	  xmlns:utu="using:Uno.Toolkit.UI"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+	  mc:Ignorable="d">
+
+	<Grid utu:SafeArea.Insets="VisibleBounds">
+		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto" />
+			<RowDefinition />
+		</Grid.RowDefinitions>
+		<utu:NavigationBar Content="Page Navigation" />
+
+		<StackPanel Grid.Row="1"
+					HorizontalAlignment="Center"
+					VerticalAlignment="Center"
+					Spacing="16">
+			<Image Width="150"
+				   Height="150"
+				   Margin="12"
+				   HorizontalAlignment="Center"
+				   Source="ms-appx:///Assets/logo.png" />
+
+			<!--  The Navigation.Request attached property is used to navigate to the SamplePage  -->
+			<Button uen:Navigation.Request="Sample" Content="Navigate via Xaml" />
+
+            <!-- New button using Navigation via code behind -->
+            <Button Command="{Binding NavigateCommand}"
+                    Content="Navigate via code behind" />
+
+            <!-- Button to show Message Dialog -->
+            <Button Content="Show Simple Message Dialog"
+                    Command="{Binding ShowSimpleDialogCommand}" />
```

Comment:
Can we have a page dedicated exclusively to Message Dialog (Xaml and Code-behind) and another one for Dialog (Modal and Flyout)?